packages/ocamlnet/ocamlnet.4.1.9-2/files/pkg-config.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/code/src/nettls-gnutls/configure b/code/src/nettls-gnutls/configure
+index 247ee47..65833db 100755
+--- a/code/src/nettls-gnutls/configure
++++ b/code/src/nettls-gnutls/configure
```

Comment:
```suggestion
--- a/src/nettls-gnutls/configure
+++ b/src/nettls-gnutls/configure
```
easy. This patch has been made from the ocamlnet source repository and it does not have the same directory structure as the archive. The archive is basically the `code/` subdirectory from the source repository

Reply:
Aaah, thanks a lot!